virttest/qemu_vm.py

```diff
@@ -2075,15 +2068,13 @@ def sort_key(dev):
                     "disable_copy_paste", "spice_seamless_migration",
                     "listening_addr"
                 )
-
```

Comment:
also style fix, please move it to a separate commit along with the docstring fix (or remove it)